TKeazirian.HTTPServer/Router/Router.cs

```diff
@@ -53,14 +58,9 @@ private bool RoutesConfigContainsPath(Request request)
         return _routesConfig.Routes.ContainsKey(request.GetRequestPath());
     }
 
-    private static bool IsOptionsRequest(Request request, Handler handler)
+    private static bool IsOptionsRequest(Request request)
     {
-        if (handler.AllowedHttpMethods().Contains("GET"))
```

Comment:
@tpetersen0308 I removed this requirement here and it is now working as expected.